### DISCUSSION TITLE
From MSFT Sculpt to Corne using Miryoku on QMK and ZMK

### DISCUSSION THREAD

**Discussion by boekhocg** [manna-harbour/miryoku#59] [Show and tell]
Wanted to show of my boards running Miryoku on QMK and ZMK. 
![CRKBD](https://user-images.githubusercontent.com/61322825/146645881-2c91502d-e3b4-4455-90db-8f81d457d849.jpg)

After some research and seeing many recommendations to Miryoku, I decided to give it a go. Getting used to home row mod, coming from standard keyboard, took a little bit getting used to. The first couple of days, I was not very productive. I used typingclub to practice and by day 3, was able to type fine. Not near as fast as I was before, but quickly improving. I've been steadily improving within a few weeks and my fingers and wrists see alot less movement.  Additionally, after 30 years of touch-typing, for the first time in my life, I am able to touchtype symbols and numbers without looking at the keyboard. With standard keyboards, even while being a software developer, I was never able to learn to accurately type numbers/symbols without looking.
Been very happy with my decision.
I'm still on QWERTY, and may at some point give Coleman DH or Workman a go, however for now, I'm sticking with QWERTY.

If you want to read more on my transitioning to Corne, you can read more here: https://www.reddit.com/r/ErgoMechKeyboards/comments/rinoig/sculpt_to_corne/
Thank you Manna-Harbour! You rule!

**Comment by manna-harbour** [comment]
Thanks so much for sharing your experience and builds. I'm really happy it's going well!
